feature/notification/src/main/java/com/sseotdabwa/buyornot/feature/notification/ui/NotificationDetailScreen.kt

```
@@ -1,78 +1,238 @@
+                                .fillMaxSize()
+                                .verticalScroll(rememberScrollState()),
+                    ) {
+                        FeedCard(
+                            modifier = Modifier.padding(20.dp),
+                            profileImageUrl = feed.author.profileImage ?: "",
+                            nickname = feed.author.nickname,
+                            category = feed.category,
+                            createdAt = TimeUtils.formatRelativeTime(feed.createdAt),
+                            content = feed.content,
+                            productImageUrl = feed.viewUrl,
+                            price = String.format("%,d", feed.price),
```

detekt (1.23.8)

[warning] String.format("%,d", feed.price) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)